packages/devtools_app/test/debugger_screen_test.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+@TestOn('browser')
```

Comment:
instead of adding this to all the tests that test something related to ansicolor outputs, can we try pulling only the affected tests into ansi_up_test.dart? you can make a group for each set of tests (e.g. 'Debugger ansi tests', 'Logging ansi tests'). This would also eliminate the need below to make logging_screen_browser_test and logging_screen_vm_test

Reply:
done